docs/source/settings.rst

```diff
@@ -142,6 +142,76 @@ field following this format:
       'spidermon_field_coverage/dict/field_1/nested_field_1_2': 0.5,
       'spidermon_item_scraped_count/dict/field_2': 0.5,
 
+SPIDERMON_FIELD_COVERAGE_SKIP_WITHOUT_FIELD_COVERAGE_RULES
```

Comment:
If we proceed, I think we need a shorter setting name.

I wonder if it would not be too messy to support a special value for `SPIDERMON_ADD_FIELD_COVERAGE`, e.g. `"rules"`.

Otherwise, what about `SPIDERMON_FIELD_COVERAGE_RULES_ONLY`, or something else similarly short?